search-app/src/cmr/search/site/routes.clj

```diff
@@ -24,22 +24,29 @@
              (pages/home ctx))
         (GET "/sitemap.xml"
              {ctx :request-context}
-             (pages/sitemap-master ctx))
-        (GET "/site/sitemap.xml"
-             {ctx :request-context}
-             (pages/sitemap-top-level ctx))
+             {:status 200
+              :body (slurp (static/site-resource "../sitemap.xml"))})
         (GET "/site/collections/directory"
              {ctx :request-context}
              (pages/collections-directory ctx))
         (GET "/site/collections/directory/eosdis"
-             {ctx :request-context}
-             (pages/eosdis-collections-directory ctx))
+             [provider-id tag]
+             (when-let [resource (static/site-resource
```

Comment:
Do we need any validation on these parameters?

Reply:
Not ignoring you :-) Will come back to this tomorrow. Today was a massive rabbit-run into bugs-in-tests-land ...

Reply:
That's a question I had asked about the routes (in general) when I first joined the CMR team. Turns out that is done deeper in the stack.

In this particular case, the user doesn't supply the values. These resources are generated statically; any provider-id or tag supplied that doesn't match a corresponding file on the file system simply returns a 404.